.php-version

```diff
@@ -0,0 +1 @@
+8.1
```

Comment:
Thank you for the Pull Request @renttek !  
I was developing this module on PHP 7.4 so not sure about the support of PHP 8.1 - have you checked it out there?  
Does it work fine?  

Thx!

Reply:
Just checked it - works fine on PHP 8.1 😅

Reply:
Oh sorry, I just took 8.1 because of the next M2 version and didn't think about it anymore.